src/eu/deltasource/internship/Account.java

```diff
@@ -0,0 +1,71 @@
+package eu.deltasource.internship;
```

Comment:
Normally you should add something else after 'eu.deltasource.internship', e.g. eu.deltasource.internship.accountapplication'